install/0000_30_machine-api-operator_09_rbac.yaml

```diff
@@ -332,6 +332,21 @@ subjects:
     name: prometheus-k8s
     namespace: openshift-monitoring
 
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: cloud-provider-configmap-reader
+  namespace: openshift-config
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: cloud-provider-configmap-reader
```

Comment:
In terms of naming `cloud-provider` is circunstancial here, this role gives you ability to read all configmaps in the namespace regardless a consumer uses it for reading the cloud-provider or something else.
Also I wonder if this should reflect ownership and be named after `machine-api-controllers-*`? cc @JoelSpeed @bison wdyt?

Also can you please add this to the related resources https://github.com/openshift/machine-api-operator/blob/1c9807c657875c76f2b0e7f93d2075cee5e6af10/pkg/operator/status.go#L182-L214?

Reply:
Is there any way to narrow things down to just the singular configmap?

Reply:
@enxebre What did you want me to add in there? 
```json
...
               {
			Group:    "rbac.authorization.k8s.io",
			Resource: "roles",
			Name:     "machine-api-controllers",
		}
}
```